firmware/integration/rusefi_config.txt

```diff
@@ -2534,10 +2534,10 @@ end_struct
 #define TS_TRIGGER_SCOPE_CHANNEL_2_NAME "Channel 2"
 
 ! some board files override this value using prepend file
-#define ts_show_ign_key_switch false
+#define ts_show_ign_key_switch true
```

Comment:
just cherry picked 5 commits out of six, not this one - most boards should NOT have these visible

problem statement: not sure if defaults should be most universal or not